src/components/HeaderBarCustomDialog.vue

```diff
@@ -198,7 +198,8 @@ watch(
     if (oldData.length < newData.length) {
       selectedButton.value = newData[newData.length - 1];
     } else if (
-      selectedButton.value !== undefined &&
+      // on -> off になったとき
```

Comment:
あ、ここは主語があった方がいいかもです！
「ツールバーボタンを追加するかどうかのスイッチが」でしょうか。

まあその文脈はここのコードから関係ない（つまりいつかコメントが嘘になってしまう可能性が高い）ので、コードの文脈に合うようにこうコメントしてあげるといいかもです！
```suggestion
      // ツールバーボタンの数が減ったとき
```

Reply:
こちらそもそも確認のための不要なコメントだったので削除しました🙇‍♂️